src/helpers/hasZeroTickets.ts

```diff
@@ -6,5 +6,6 @@ import {
 const ZERO = BigInt.fromI32(0)
 
 export function hasZeroTickets(player: Player): boolean {
-  return player.consolidatedBalance.plus(player.latestBalance).equals(ZERO)
+  // return player.consolidatedBalance.plus(player.latestBalance).equals(ZERO)
```

Comment:
You can delete this line. This is a leftover from v2

Reply:
Prob can delete this entire file, since it's easy to read the code for zero tickets now.

Reply:
done!